### PR TITLE
created routes for flask app

### PR DESCRIPTION
**Pull request by WilliamSterner**
Added routes for flask app in recipe

### CODE REVIEW

File: backend/main.py [hunk 1]
```diff
@@ -10,61 +10,92 @@
...
+    Recipe.addRecipe(id, recipes)
     return jsonify({'message': 'Recipe created successfully'})
 
 
 @recipeAPI.route('/api/removeRecipe', methods=['POST'])
 def delete_recipe():
     data = request.get_json()
+    id = data['id']
     name = data['name']
     ingredients = data['ingredients']
```
Comment (Grayson34) [inline backend/main.py:30]: dont need ingredients/measurements/steps

File: backend/main.py [hunk 2]
```diff
@@ -10,61 +10,92 @@
...
-    recipes = recipeBook.get_recipes()
-    recipe_data = [{'name': recipe.get_name(), 'ingredients': recipe.get_ingredients()} for recipe in recipes]
-    return jsonify(recipe_data)
+    data = request.get_json()
+    id = data['id']
+    return Recipe.getRecipes(id)
+    
 
 
 @recipeAPI.route('/api/addIngredient', methods=['POST'])
```
Comment (Grayson34) [inline backend/main.py:48]: change route to include either stock or grocery list so we know what we are adding ingredient to

File: backend/main.py [hunk 3]
```diff
@@ -10,61 +10,92 @@
...
-    ingredient = data['ingredient']
+    id = data['id']
+    item = data['item']
 
-    groceryList.add_item(ingredient)
+    GroceryList.add_item(id, item)
     return jsonify({'message': 'Ingredient added successfully'})
 
 
 @recipeAPI.route('/api/removeIngredient', methods=['POST'])
```
Comment (Grayson34) [inline backend/main.py:58]: same as above, better route name

**Review by cdewitt02 — COMMENTED** [review]
We should really consider using GET when we can to adhere to HTTP request standards. Additionally, GET requests can often be cached by the browser, thus improving performance

**Review by ryanmercado — APPROVED** [review]
Looks good, may be worth it to use GET instead of POST. Can be changed later

File: backend/main.py [hunk 4]
```diff
@@ -10,61 +10,87 @@
 @recipeAPI.route('/api/addRecipe', methods=['POST'])
 def create_recipe():
     data = request.get_json()
+    id = data['id']
```
Comment (Grayson34) [inline backend/main.py:13]: these "id"'s will be called "usr_id" now, so this needs to change throughout the file

**Review by Grayson34 — APPROVED** [review]
looks good

**Pull request merged**